treeherder/model/models.py

```diff
@@ -189,7 +189,8 @@ class Bugscache(models.Model):
     crash_signature = models.TextField(blank=True)
     keywords = models.TextField(blank=True)
     os = models.CharField(max_length=64, blank=True)
-    modified = models.DateTimeField()
+    modified = models.DateTimeField(db_index=True)
```

Comment:
Looking at the other branch I think this index is for use by this?
https://github.com/mozilla/treeherder/compare/master...sarah-clements:intermittents_view#diff-0583559162a4f446e1724a7e747802e9R27

I think that query needs to be filtering the date of the job (ie when the failure occurred), rather than the last time the bug was modified in bugscache? (bugscache is a copy of what's held in bugzilla, so the modified time corresponds to whenever someone left a comment or added themselves to CC etc)

Reply:
To clarify - this index might not be needed, if the `Bugscache.objects.filter(modified__range=(startday, endday))` is changed to use the job start/finish date instead.

Reply:
I thought about adding that in from Bugzilla but noticed that created time is used by the elastic search class (in classification_mirroring) as a classification_timestamp and start/end dates are grabbed from the Job class. But yeah, makes more sense to filter by a different date.